clang/test/Driver/sycl-no-rdc-fat-archive.cpp

```diff
@@ -1,4 +1,6 @@
 /// test behaviors of passing a fat static lib with -fno-sycl-rdc
+// UNSUPPORTED: system-windows
```

Comment:
I think that's ok as a quick fix for now to make post-commit green, but I wonder if we need a follow-up PR with tests for the flag for Windows

Reply:
Yeah I am going to investigate the actual issue after this